e2e/e2e.spec.ts

```diff
@@ -466,8 +469,14 @@ describe("e2e tests", () => {
     const installationId = fakeGitHub.mockAppInstallation(testOrg, repo);
     fakeGitHub.mockAppInstallation(testOrg, "bazel-central-registry");
 
-    const releaseArchive1 = await makeReleaseTarball(repo, "module-1.0.0");
-    const releaseArchive2 = await makeReleaseTarball(repo, "submodule-1.0.0");
+    const releaseArchive1 = await makeReleaseTarball(
+      repo,
+      "multi-module-1.0.0"
+    );
+    const releaseArchive2 = await makeReleaseTarball(
+      repo,
+      "multi-module-1.0.0"
+    );
```

Comment:
These were incorrect. There was a bug in the strip prefix checking that is fixed in this PR.